modules/neuroimaging/denoising/nlmeans/main.nf

```diff
@@ -0,0 +1,51 @@
+
+process DENOISING_NLMEANS {
+    tag "$meta.id"
+    label 'process_single'
+
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://scil.usherbrooke.ca/containers/scilus_2.0.0.sif':
+        'scilus/scilus:2.0.0' }"
+
+    input:
+    tuple val(meta), path(image), path(mask)
+
+    output:
+    tuple val(meta), path("*_denoised.nii.gz")      , emit: image
+    path "versions.yml"                             , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def prefix = task.ext.prefix ?: "${meta.id}"
+    def args = []
+    if (mask) args += ["--mask $mask"]
+
+    """
+    export ITK_GLOBAL_DEFAULT_NUMBER_OF_THREADS=1
+    export OMP_NUM_THREADS=1
+    export OPENBLAS_NUM_THREADS=1
```

Comment:
Is this always supposed to be `1`? Otherwise you could access the amount of threads available to the process using `task.cpus`